### PR TITLE
chore: add office hour blog post 28-07

### PR DESCRIPTION
**Pull request by tomaszbarwicki**
Fixes https://github.com/eclipse-tractusx/sig-infra/issues/203
Updates https://github.com/eclipse-tractusx/sig-infra/issues/93

### CODE REVIEW

File: blog/2023-07-28-OfficeHours.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Veracode token expired. The token is used for GitHub Actions. Upload and scan feature doesn't work currently, expected to fix within a week.
+- Request to check Veracode finding since many SCA findings are present. It's recommended to update dependencies.
+
+### FOSS
+
+- No presence.
+
+## Open discussions
+
+- Question arised about reviewing open IP issue for Digital Twin Registry application. Advised that Eclipse Foundation needs to review that, as it's not DevOps scope. With regards to PR review, it was stated that every committer can review PR. In general two step approach is followed, first legal stuff is verified and then application related by a team member.
```
Comment (FaGru3n) [inline blog/2023-07-28-OfficeHours.md:38]: Question `arised`  --> Question `raised`

**Review by FaGru3n — CHANGES_REQUESTED** [review]
litte typo

**Review by FaGru3n — APPROVED** [review]
LGTM

File: blog/2023-07-28-OfficeHours.md (hunk 1 above)
Comment (tomaszbarwicki) [reply blog/2023-07-28-OfficeHours.md]: Fixed.

**Review by FaGru3n — APPROVED** [review]
LGTM

**Pull request merged**